blocks/v2-hero/v2-hero.css

```diff
@@ -0,0 +1,82 @@
+/* stylelint-disable selector-class-pattern */
+.v2-hero {
+  position: relative;
+
+  /* height: 798px; */
+  height: 588px;
```

Comment:
I’m not a fan of setting absolute `height` in places like this. Need to think about the value, but could we work with the `vh` unit here?

Reply:
I changed it to `100vh`. We could change it to `calc(100vh -  var(--nav-height))` ?

Reply:
Yes, good suggestion!